### DISCUSSION TITLE
winget vs PowerShell install

### DISCUSSION THREAD

**Discussion by poiug07** [JanDeDobbeleer/oh-my-posh#1059] [Troubleshoot]
I use powershell most of the time, so appearance of other shells, doesn't matter for me. Except WSL, but I don't want WSL and PowerShell look same, therefore I go with pure WSL. I have 2 install options, using winget or using PowerShell Module. Which one is better?

**Comment by JanDeDobbeleer** [comment]
If you ONLY plan to use oh-my-posh from Powershell, the module is the easiest way.
